.github/workflows/stale.yml

```diff
@@ -0,0 +1,61 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This workflow labels and then closes stale PRs that haven't seen attention
+# for several months.
+
+name: Stale PRs
+on:
+  schedule:
+    #- cron: '00 6 * * *' # Run once a day at 6:15 UTC
+    - cron: '0 */6 * * *' # Initially we run every 6 hours until we have reached steady state
+  workflow_dispatch:
+    inputs:
+      operationsPerRun:
+        description: 'Max GitHub API operations'
+        required: true
+        default: 200 # Initially we set this at a high level to clear the backlog
```

Comment:
can we use variables at the top for all numbers?

Reply:
This is a variable, its referenced in the action.